app/views/contexts/questionnaire_summary_context.py

```diff
@@ -19,17 +19,25 @@ def __call__(self, collapsible=True, answers_are_editable=True):
 
     def _build_all_groups(self):
         """ NB: Does not support repeating sections """
-        section_summary_context = SectionSummaryContext(
-            self._language,
-            self._schema,
-            self._answer_store,
-            self._list_store,
-            self._progress_store,
-            self._metadata,
-        )
-
         for section_id in self._router.enabled_section_ids:
-            for group in section_summary_context(Location(section_id=section_id))[
-                "summary"
-            ]["groups"]:
-                yield group
+            section = self._schema.get_section(section_id)
+            if section.get("summary"):
```

Comment:
I think this should this be checking the summary `items` property. Currently any section with a section summary (custom or not) won't be on the questionnaire summary?